cni-plugins/sriov-passthrough-cni/deploy/multus-cni.yaml

```diff
@@ -0,0 +1,183 @@
+---
```

Comment:
Weird that we ship multus as a part of a CNI plugin. I would expect this to have a separate subdir under cni-plugins.

Also, this is not needed on OpenShift 4, what provider are we using?